pyproject.toml

```diff
@@ -12,14 +12,11 @@ authors = [{name = "Microsoft", email = "graphtooling+python@microsoft.com"}]
 description = "The Microsoft Graph Python SDK"
 dependencies = [
     "azure-identity >=1.12.0",
-    "microsoft-kiota-abstractions >=1.3.0,<2.0.0",
-    "microsoft-kiota-authentication-azure >=1.0.0,<2.0.0",
-    "microsoft-kiota-serialization-json >=1.3.0,<2.0.0",
-    "microsoft-kiota-serialization-text >=1.0.0,<2.0.0",
-    "microsoft-kiota-serialization-form >=0.1.0",
-    "microsoft-kiota-serialization-multipart >=0.1.0",
-    "microsoft-kiota-http >=1.0.0,<2.0.0",
-    "msgraph_core >=1.0.0"
+    "microsoft-kiota-serialization-json >=1.8.0,<2.0.0",
```

Comment:
can you please provide more context on why we're removing azure, abstractions and http here please?

Reply:
Because they are part of msgraph-core, by not having them here as well, we don't have to keep them updated in two repos.

Reply:
thanks for the context, I wanted to make sure it's captured.